assembly/cli/src/main/java/org/eclipse/sw360/antenna/frontend/cli/AntennaCLIOptions.java

```diff
@@ -54,6 +58,27 @@
     AntennaCLIOptions(String configFilePath, boolean debugLog, boolean showHelp, boolean valid) {
         super(debugLog, showHelp, valid);
         this.configFilePath = configFilePath;
+        this.propertiesFilePath = null;
```

Comment:
This constructor should simply call the other one with a *null* parameter for the properties path.